services/movie.py

```diff
@@ -0,0 +1,50 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,

```

Comment:
If argument can be `None` you must show it in type hint. Fix everywhere.
```suggestion
        genres_ids: list[int] | None = None,

```